### PR TITLE
ENH: Revise and add tests for the PEPOLAR correction

### PR DESCRIPTION
**Pull request by oesteban**

### CODE REVIEW

**Comment by oesteban** [comment]
I still need to double-check the simplified workflow is equivalent to the original one, but I'm adding Matt since the review can be started.

**Comment by oesteban** [comment]
Hey @mattcieslak, can you give this a quick look?

I'll be addressing the suggestions you gave under #18 in a different PR (the reason is that there are some more hard-wired elements specific to BOLD, so such generalization will require more surgery on these workflows)

Results are here:

https://181-189236569-gh.circle-artifacts.com/0/tmp/tests/sdcflows/sub-1/fmap/sub-1_dir-AP_desc-pepolar_epi.svg
https://181-189236569-gh.circle-artifacts.com/0/tmp/tests/sdcflows/sub-HCP101006/fmap/sub-HCP101006_dir-LR_desc-pepolar_epi.svg

BTW, what do you think about the new way of plotting field maps (bilateral with linear intensity gradient scheme)?

File: sdcflows/workflows/tests/test_pepolar.py [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+def test_pepolar_wf1(bids_layouts, output_path, dataset, workdir):
+    """Test preparation workflow."""
+    layout = bids_layouts[dataset]
+
+    if dataset == 'testdata':
+        bold = layout.get(suffix='bold', dir='LR', direction='LR',
+                          extension=['.nii.gz', '.nii'])[0]
+        boldref = layout.get(suffix='boldref', dir='LR', direction='LR', desc='brain',
+                             extension=['.nii.gz', '.nii'])[0]
+    elif dataset == 'ds001600':
```
Comment (mattcieslak) [inline sdcflows/workflows/tests/test_pepolar.py]: Does this work? There are lots of bold datasets in here, each with a different fieldmap

**Review by mattcieslak — APPROVED** [review]
I like the refactor! I'm also working on tests for ds001600 at the moment. The new fieldmap plots also look great. Does the color scheme reflect displacements in RAS+?

**Comment by oesteban** [comment]
The color scheme does not have into account the polarity of the phase encoding, so unfortunately not yet.

It is meant to be useful in the representation of fieldmaps in Hz (e.g. https://181-189236569-gh.circle-artifacts.com/0/tmp/tests/sdcflows/sub-1/fmap/sub-1_acq-v4_phasediff.svg and https://181-189236569-gh.circle-artifacts.com/0/tmp/tests/sdcflows/sub-HCP101006/fmap/sub-HCP101006_phasediff.svg).

**Pull request merged**